src/test/resources/features/convertVideo/video.feature

```diff
@@ -0,0 +1,85 @@
+Feature: Video Controller
+
+  In order to convert a file
+  As a valid Converter customer
+
+  Background: Sets authentication
+    Given I set valid authentication headers
+  @functional
+  Scenario: Verify that "/convertVideo" can perform "POST" request converting into MP4
```

Comment:
Please consider improving this title to something like: `Verify that is possible to convert a video to MP4 format`

Reply:
I modified it according your suggestion thanks.